Sprint-1/stretch/aoc-2018-day1/solution.js

```diff
@@ -0,0 +1,22 @@
+
+const fs = require('fs');
+
+fs.readFile('/root/CYF/Module-Data-Groups/Sprint-1/stretch/aoc-2018-day1/input.txt', (err, data) => {
+  if (err) throw err;
+  console.log(device(data.toString().split("\n")));
+});
+
+function device(array){
+    let result = 0;
+    for(let i =0; i < array.length; i++){
+        if(array[i].slice(0, 1) == "+"){
```

Comment:
To test if the first character is a "+", you could also write `array[i][0] == "+"` or `array[0].charAt(0) == "+"`.

You can possibly simplify your code and make your code perform better.
Hint: What are the value of `Number("+10")` and `Number("-10")`?